src/components/20-molecules/dropdown/index.js

```diff
@@ -356,7 +329,6 @@ class AXADropdown extends NoShadowDOM {
   firstUpdated() {
     // dropdowns cannot meaningfully be initially open, since the underlying
     // <select> cannot be force-opened natively
-    this.open = false;
     this.dropdown = this.querySelector('.js-dropdown__content');
     this.select = this.querySelector('.js-dropdown__select');
     this.addEventListener('focus', this.onFocus, false);
```

Comment:
Why not bind focus and blur eventListener directly in template? Then there no need to clean them on disconnected callback.

Reply:
done